components/whyd/2022/Terminal.vue

```diff
@@ -135,4 +156,8 @@ p.error-text {
 p.italic-text {
   font-style: italic;
 }
+
+p.confirmed-text {
+  color: lime;
```

Comment:
Color nit here too: same with the comments on `red` I'd reduce the saturation or luminosity of this color a bit, and factor it into a css var

Reply:
reduced saturation slightly, can take another pass later as needed.